### DISCUSSION TITLE
用户输入的内容能否加个按钮，让内容直接 源码 显示？

### DISCUSSION THREAD

**Discussion by se7enmuting** [lobehub/lobehub#1340] [General]
用户输入的内容默认也是 markdown 显示，但平时一般简单的内容代码就是随手复制粘帖，也懒得在前后加代码标识： ```
然后显示就会很怪异，大写，斜体，变色等等，见下图：
![image](https://github.com/lobehub/lobe-chat/assets/889925/40423801-25f5-444d-91f8-e09b40376c41)
![image](https://github.com/lobehub/lobe-chat/assets/889925/a930feab-9801-44c6-91c1-c3a37fc34596)
能否加个按钮，显示源码模式，或者加个设置，希望考虑下
在 Discussion 和 Issues 里搜了 源码，好像并没有相关内容，所以这里提一下

**Comment by arvinxx** [comment]
@canisminor1990 这个好搞么？
